### PR TITLE
adding yml for byteguard-build-actions plugin

### PR DESCRIPTION
**Pull request by Khushboo-Baheti**
# Description
https://github.com/jenkinsci/byteguard-build-actions-plugin
https://issues.jenkins-ci.org/browse/HOSTING-627

Following users should have permission:
@khushboo(Artifactory)
@victor9000(Artifactory)

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by Khushboo-Baheti** [comment]
Thanks. can you please merge my pull request.

**Comment by daniel-beck** [comment]
This is a new plugin. Please review the checklist and perform all applicable actions.

**Comment by Khushboo-Baheti** [comment]
Hello, 
I have reviewed the checklist and perform all applicable actions.Please merge my pull request.

File: permissions/plugin-byteguard-build-actions.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "byteguard-build-actions"
+github: "jenkinsci/byteguard-build-actions-plugin"
+paths: 
+- "org/jenkins-ci/plugins/byteguard-build-actions"
```
Comment (daniel-beck) [inline permissions/plugin-byteguard-build-actions.yml]: Mismatch to POM contents. Please review this repo's readme what is required here.

**Pull request merged**